api/hearings/data/hearing-actuals.mock.data.ts

```diff
@@ -8,7 +8,7 @@ export const HEARING_ACTUAL: HearingActualsMainModel = {
       hearingResult: HearingResult.CANCELLED,
       hearingResultDate: '2019-01-01',
       hearingResultReasonType: '',
-      hearingType: 'Pre-hearing review',
+      hearingType: 'Final',
```

Comment:
could this map with the key of the ref data, so this case it should be 'final'.

Reply:
Thank you @rayliangatos - I have now updated the value to be 'final'